Automattic/newspack-sponsors
# fix: update composer file to correct linting error

### All Submissions:


### Changes proposed in this Pull Request:

This PR updates composer.json to allow `dealerdirect/phpcodesniffer-composer-installer`, which should correct the linting error caused by the last release merge.

I don't actually get the same issue locally when installing composer, so this may not be the correct way to fix!

### How to test the changes in this Pull Request:

1. Confirm that `composer install` runs well
2. Confirm that this PR doesn't trip the linting in Circle CI 🙂 

### Other information:
